### DISCUSSION TITLE
remove contributors

### DISCUSSION THREAD

**Discussion by joyalb9** [community/community#53680] [New to GitHub]
i did a commit but it shows my frnd has done commit how do i remove her from my contribution its my personal project

**Comment by airtower-luna** [comment]
GitHub assigns commits to users based on the *author email* recorded in the commit. Make sure your [user name and email are set correctly](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup#_your_identity), for GitHub to recognize your commits `user.email` must be an address connected to your account.

To fix the author email in past commits you need to rewrite the repository history and change the commits with the wrong author email. [Git Tools - Rewriting History](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) describes how to do that. Note that this will change *all* commit IDs after the changed commits. This means anyone who already has a copy of your repository and tries to fetch updates will have to manually accept the modified history. :warning:
